### PR TITLE
Update contributing guide

### PR DESCRIPTION
**Pull request by robb-romans**
- add triangular workflow instructions
- detailed git config
- master branch pre-commit hook
- don't use gerunds for section titles
- revise image instructions
- fix syntax highlighting
- rationalize heading levels
- remove suggested topics

### CODE REVIEW

**Comment by robb-romans** [comment]
Click here to view the rendered file https://github.com/rackerlabs/technical-blog/blob/edd14e63f9faf1b0245e4a57777296bab26f745c/CONTRIBUTING.md

**Comment by WilliamLoy** [comment]
@catlook would you mind reviewing the sections I added?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -7,39 +7,91 @@ out of applications and platforms. Many also add a personal perspective to the
...
-4. Within the newly created folder, create a new file with the following naming convention:
-   `index.md`.The published URL for your post becomes `https://docs.rackspace.com/blog/title-of-your-post/` when published.
+      ```bash
+      git checkout -b name-of-your-branch
+      ```
+1. Navigate to the root directory of your local copy of this repo using the command line.
 
-**Note:** If your post has images, you should place the image files within the newly created
-folder.
+1. Create a new directory within ```/content/blog/YYYY/``` that contains the inital ```.index.md```
```
Comment (robb-romans) [inline CONTRIBUTING.md:83]: @WilliamLoy put the prerequisite to install Hugo here

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -7,39 +7,91 @@ out of applications and platforms. Many also add a personal perspective to the
...
+      git checkout -b name-of-your-branch
+      ```
+1. Navigate to the root directory of your local copy of this repo using the command line.
 
-**Note:** If your post has images, you should place the image files within the newly created
-folder.
+1. Create a new directory within ```/content/blog/YYYY/``` that contains the inital ```.index.md``` 
+   by using the following command:
+  
+   ```hugo new --kind blog-bundle blog/YYYY-MM-DD-title-of-post```
```
Comment (robb-romans) [inline CONTRIBUTING.md:86]: @WilliamLoy please change this to hugo new --kind blog-bundle blog/YYYY/YYYY-MM-DD-title-of-post

**Comment by robb-romans** [comment]
Thanks all! If you have questions or see problems in the merged file, open a PR to fix or an issue to discuss.

**Pull request merged**